lib/matplotlib/colorbar.py

```diff
@@ -1155,6 +1155,16 @@ def remove(self):
         self.ax.remove()
 
 
+def _merge_disjoint_kwargs(d, **kwargs):
+    for k, v in kwargs.items():
```

Comment:
Should get a docstring. In particular, it should mention that `d` is a dict and is updated with `kwargs`. Also, I'd rather use `_add_disjoint_kwargs`. "merge" sounds more like it would return a new dict with the merged content.

Reply:
sure, done